setup_venv.ps1

```diff
@@ -36,6 +53,8 @@ Write-Host $p
 Write-Host "Installing Build Dependencies"
 python -m venv .\shark.venv\
 .\shark.venv\Scripts\activate
+python -m pip install --upgrade pip
+pip install wheel
```

Comment:
I would also just remove iree-compiler and runtime here too so it always reinstalls when you run setup_venv.ps1. But can happen later

Reply:
I would like to redo this whole script anyways being object oriented with try-catch phrases and help functions. Also a what-if and verbose is missing.
But that needs some time